docs/dev/testing.md

```diff
@@ -302,6 +302,63 @@ suite is as solid as possible. The first step is contributing a stable (read: no
 To do so, when developing tests, please run them (1) in debug mode and (2) 100 times in a row (using `--repeat 100`),
 and see that they pass successfully.
 
+## Allure reporting
+
+To make analyzing of the test results, a more convenient allure reporting tool is introduced.
+Python module allure-pytest module is included in the toolchain image and additional parameter added to gather allure
+data for the test run.
+However, the allure binary is not a part of the toolchain image.
+So to have the full benefit of the new reporting tool allure binary should be installed locally.
+### Allure installation
+Allure offers deb and rpm packages and binary: https://github.com/allure-framework/allure2/releases
+Download allure according to your system: deb package for Ubuntu and rpm for Fedora.
+
+Install downloaded package with next command:
+Ubuntu:
+```shell
+sudo apt install path_to_deb_package
+```
+Fedora:
+```shell
+sudo dnf install path_to_rpm_package
+```
+**Note:** rpm package requires dependency `default-jre-headless` but on Fedora 38 none of the packages provides it.
+So manual installation is required.
+
+#### Manual allure installation
+Allure provides the package with all dependencies in tgz archive.
+This package can be used on Ubuntu or Fedora if package cannot be used for any reason.
+
+The third party programs usually installed in the `/opt` directory on unix based systems.
+This is requiring the root permissions.
+Another location where allure can be placed is a `~/.local/share` directory that is not requiring the root permissions
+since it's a home folder of the current user.
+
+1. Download the tgz archive from the [release page](https://github.com/allure-framework/allure2/releases)
+2. Create destination directory where the allure binary will be located:\
+```shell
+$ sudo mkdir /opt/allure
+```
+3. Extract the tgz archive to the desired destination with a command:\
+```shell
+$ sudo tar -xf allure-2.29.0.tgz --directory /opt/allure --strip-components=1
+```
+4. Add allure binary to you PATH directly from `/opt` directory or add symlink to `~/.local/bin` which should be already in the PATH\
+```shell
+$ ln -s /opt/allure/bin/allure ~/.local/bin/allure
+```
+
+### Basic allure usage
+1. Open directory with the test results
```

Comment:
This is ambiguous. Is this `testlog`? Or do I need to go deeper into a specific mode or test run?

Reply:
Added more information to make it less ambiguous.